Sources/Testing/Testing.docc/OrganizingTests.md

```diff
@@ -82,47 +82,47 @@ Are equivalent to:
 }
 ```
 
-## Constraints on test suite types
+### Constraints on test suite types
 
-If a type is used as a test suite, it is subject to some constraints that are
+When using a type as a test suite, it's subject to some constraints that are
 not otherwise applied to Swift types.
 
-### An initializer may be required
+#### An initializer may be required
 
 If a type contains test functions declared as instance methods, it must be
 possible to initialize an instance of the type with a zero-argument initializer.
 The initializer may be any combination of:
 
-- implicit or explicit;
-- synchronous or asynchronous;
-- throwing or non-throwing; and
-- `private`, `fileprivate`, `internal`, `package`, or `public`.
+- implicit or explicit
+- synchronous or asynchronous
+- throwing or non-throwing
+- `private`, `fileprivate`, `internal`, `package`, or `public`
 
 For example:
 
 ```swift
 @Suite struct FoodTruckTests {
   var batteryLevel = 100
 
-  @Test func foodTruckExists() { ... } // ✅ OK: type has implicit init()
+  @Test func foodTruckExists() { ... } // ✅ OK: The type has an implicit init().
```

Comment:
You added punctuation here, but not in the examples for `@Tag`.

Reply:
Missed those; will go back and do those. Thanks for catching that.

Reply:
Fixed in this commit: `0bafd07613dc8ab33a45f8c61f8d0986d0db5d03`